k3d/README.md

```diff
@@ -0,0 +1,72 @@
+# K3d (Kubernetes in Docker)
+
+run K8s clusters locally with an image registry `localhost:5000`
+
+```shell
+# the shell to immediately exit if any command encounters an execution error.
+set -o errexit 
+
+IMAGE=${IMAGE:rancher/k3s:v1.25.2-k3s1}
+
+# create registry container unless it already exists
+reg_name='k3d-registry'
+reg_port='5000'
+cluster_name='k3d'
+if ! k3d registry list | grep -q "${reg_name}"; then
+  k3d registry create "${reg_name}" --port "${reg_port}"
```

Comment:
i realize, suddenly, that we need provides another Markdown block.
The block contains command guides to install the kind/k3d cli tools :)